plugins/plugin-keycloak/che-plugin-keycloak-server/src/main/java/org/eclipse/che/keycloak/server/KeycloakAuthenticationFilter.java

```diff
@@ -42,7 +40,7 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-public class KeycloakAuthenticationFilter implements Filter {
+public class KeycloakAuthenticationFilter extends AbstractKeycloakFilter {
```

Comment:
Should be `@Singleton` too (it's my mistake probably)

Reply:
Fixed